packages/cadl-ranch-specs/http/type/property/optionality/main.tsp

```diff
@@ -112,19 +112,19 @@ model DurationProperty is ModelTemplate<duration>;
 @operationGroup
 interface Duration extends OperationsTemplate<DurationProperty, "\"P123DT22H14M12.011S\""> {}
 
-// Model with optional plaindate property
-@doc("Model with a plaindate property")
-model PlaindateProperty is ModelTemplate<plainDate>;
-@route("/plaindate")
+// Model with optional plainDate property
+@doc("Model with a plainDate property")
+model PlainDateProperty is ModelTemplate<plainDate>;
+@route("/plainDate")
 @operationGroup
-interface Plaindate extends OperationsTemplate<PlaindateProperty, "\"2022-12-12\""> {}
+interface PlainDate extends OperationsTemplate<PlainDateProperty, "\"2022-12-12\""> {}
```

Comment:
PlainDate is not in the format of "2022-12-12", it should be "Mon Dec 12 2022", see https://typespec.io/docs/standard-library/built-in-data-types#plainDate

Reply:
@timotheeguerin Could you help confirm here? I notice that you have examples for plainDate here https://github.com/microsoft/typespec/blob/main/packages/compiler/lib/intrinsics.tsp#L107 and it says that Create a plain date from an ISO 8601 string, and according to https://ijmacd.github.io/rfc3339-iso8601/ 2022-12-12 is a valid plainDate string and April 10th is not ?

Reply:
Yeah plainDate and plaintive should be in the iso format over json by default. Should update the encoding doc to include that